### PR TITLE
Improved usecontext@D5_O-uElftYGQr_bTU_se.md

### PR DESCRIPTION
**Pull request by SatyamVyas04**
- Rectified bullet point structure (earlier erroneous)
- Added a video tutorial for the same

### CODE REVIEW

**Comment by SatyamVyas04** [comment]
@dansholds let me know if something is to be changed 👍🏻

File: src/data/roadmaps/react/content/usecontext@D5_O-uElftYGQr_bTU_se.md [hunk 1]
```diff
@@ -5,5 +5,6 @@ This is particularly useful when many components need to access the same data or
 
 Visit the following resources to learn more:
 
--[@official@Using the useContext](https://react.dev/reference/react/useContext)
--[@official@Understand the concept of context](https://react.dev/learn/passing-data-deeply-with-context)
+- [@official@Using useContext](https://react.dev/reference/react/useContext)
+- [@official@Explore the concept of context](https://react.dev/learn/passing-data-deeply-with-context)
+- [@video@WebDevSimplified](https://www.youtube.com/watch?v=5LrDIWkK_Bc)
```
Comment (dansholds) [inline src/data/roadmaps/react/content/usecontext@D5_O-uElftYGQr_bTU_se.md:10]:
```suggestion
- [@video@Learn useContext In 13 Minutes](https://www.youtube.com/watch?v=5LrDIWkK_Bc)
```

**Comment by dansholds** [comment]
> @dansholds let me know if something is to be changed 👍🏻

FYI I'm not merging on weekends 😂

**Comment by SatyamVyas04** [comment]
> > @dansholds let me know if something is to be changed 👍🏻
> 
> FYI I'm not merging on weekends 😂

ohhhh my bad 😄

**Pull request merged**